pkgs/games/torus-trooper/default.nix

```diff
@@ -0,0 +1,106 @@
+{ lib
+, stdenv
+, fetchpatch
+, fetchurl
+, unzip
+, gdc
+, SDL
+, SDL_mixer
+, bulletml
+}:
+
+let
+debianPatch = patchname: hash: fetchpatch {
+  name = "${patchname}.patch";
+  url = "https://sources.debian.org/data/main/t/torus-trooper/0.22.dfsg1-12/debian/patches/${patchname}.patch";
+  sha256 = hash;
+};
+
+in stdenv.mkDerivation {
+  pname = "torus-trooper";
+  version = "0.22";
+
+  src =
+    (fetchurl {
+      url = "http://abagames.sakura.ne.jp/windows/tt0_22.zip";
```

Comment:
https://abagames.sakura.ne.jp/windows/tt0_22.zip returns a 404 for me.

Reply:
Weird. it doesn't for me, and neither for archive.org and ofborg... but it does for archive.today! I wonder what's the difference

Reply:
Do I have to act on this? Maybe add a mirror?

Reply:
I think it only 404 on https 😂

Reply:
That I can confirm. The `url` is http though. Is that not working for you?

Reply:
Yeah that works but https everywhere did not like it.

Reply:
I wrote them an email, and I'll update the package when I hear back from them. We have the hash, so this isn't a blocker, right?